test-plone-4.3.x.cfg

```diff
@@ -1,7 +1,13 @@
 [buildout]
 extends =
     https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-4.3.x.cfg
-    versions.cfg
     sources.cfg
 
 package-name = ftw.inflator
+
+eggs +=
+    ftw.inflator[plone4_multilingual]
+
+[versions]
+plone.app.multilingual = <2a
+archetypes.multilingual = <2a
```

Comment:
Should we move that to https://github.com/4teamwork/ftw-buildouts/blob/master/test-versions-plone-4.cfg ?

Reply:
moved to https://github.com/4teamwork/ftw-buildouts/pull/148

Reply:
👍 merged